### ISSUE TITLE
Custom Control Types for Element Studio

### ISSUE THREAD

**Issue by flowww-dev** [soflyy/breakdance-bugs#768]
It would be very cool, if it is possible to add Custom Control Types for Element Studio, so the community could extend Breakdance with custom control types.

**Comment by Bobolinos** [comment]
I need this too!

**Comment by louisreingold** [comment]
This would unlock a lot of UI possibilities.

**Comment by flowww-dev** [comment]
Yes of course - it is difficult to map custom elements as vue components. We need dynamic vue compnents.
